modules/st2-elastic-field/directive.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+angular.module('main')
+  .directive('st2ElasticField', [
```

Comment:
There is probably none, it is just my fear that with the number of modules we have we may want to start writing our very own package manager by that point =) Just trying to avoid creating new modules, at least until we get rid of ones we don't need anymore.